src/pages/DetailsPage.js

```diff
@@ -98,11 +100,25 @@ const DetailsPage = (props) => {
                 {details ? (
                     <ScrollView>
                         <View style={styles.pageWrapper}>
-                            <Avatar
-                                containerStyles={[styles.avatarLarge, styles.mb3]}
-                                imageStyles={[styles.avatarLarge]}
-                                source={details.avatar}
-                            />
+                            <PreviewAttachmentModal
+                                sourceURL={details.avatarHighResolution}
```

Comment:
Instead of this directly use `sourceURL={details.avatar || details.avatarThumbnail}` here